src/chart/bar/BarChart.tsx

```diff
@@ -203,80 +236,174 @@ const NeoBarChart = (props: ChartProps) => {
   };
 
   // Fixing canvas bug, from https://github.com/plouc/nivo/issues/2162
+  // SVGGraphicsElement.getBBox
   HTMLCanvasElement.prototype.getBBox = function tooltipMapper() {
     return { width: this.offsetWidth, height: this.offsetHeight };
   };
 
+  // positionLabel == 'off' ? {} :
   const extraProperties = positionLabel == 'off' ? {} : { barComponent: BarComponent };
   const canvas = data.length > 30;
   const BarChartComponent = canvas ? ResponsiveBarCanvas : ResponsiveBar;
+
```

Comment:
somewhere here we have changed the default behaviour of the bar chart.
It used to auto-scale to available width and height, and I think we should keep that behaviour - only use custom bar dimensions if the user has specified it explictly.

Can you make it so the "default" behaviour of the bar chart (without advanced settings) functions as before?